lldb/include/lldb/Target/StopInfo.h

```diff
@@ -79,6 +79,11 @@ class StopInfo : public std::enable_shared_from_this<StopInfo> {
 
   virtual bool IsValidForOperatingSystemThread(Thread &thread) { return true; }
 
+  /// A Continue operation can result in a false stop event
+  /// before any execution has happened in certain cases on Darwin.
+  /// We need to silently continue again time.
+  virtual bool IsContinueInterrupted(Thread &thread) { return false; }
```

Comment:
This may be a little picky, but the continue that was interrupted is in the past by the time you call this.  WasContinueInterrupted reads more naturally to me because of that.